docs/src/main/asciidoc/amazon-lambda.adoc

```diff
@@ -123,7 +123,7 @@ Build the project using maven.
 
 or, if using Gradle:
 ----
-./gradlew clean assemble
+./gradlew clean quarkusBuild --uber-jar
```

Comment:
FYI - I put gradle "assemble" out of concerns,  that #7909 had introduced a NPE when running the tests, and gradle would abort (Maven ignores it hence the Integration Tests are not picking it up).

My view was that a simple compile was required at this point, but I do agree we would want the uber jar built and ready for deployment at this point.  I believe the property quarkus.package.uber-jar=true only works with Maven? If it used this property in Gradle so you could do "gradle build" it would be cleaner. Worth validating.

Reply:
Ok, I've validated this, yes the uber jar property works with Gradle now also.
`quarkus.package.uber-jar=true
`
Can you please add that to the src/main/resources/application.properties?

You no longer need to make change this, i.e. recommend it stays at assemble as it's a standard simple command.  I've tested toggling the application.properties with true & false and validated that it does respect it.
`./gradlew clean assemble`

Reply:
Might be worth updating the Quarkus Gradle Guide with this detail also (application.properties)?  Recommend you also squash your commits into one, as you'll be asked to do that before merge.

Reply:
Also, can you update the amazon-lambda.adoc to remove the note to generate the gradle build via code.quarkus.io since this is now part of the Maven archetype?  Perhaps reword that test to indicate that the archetype includes the Gradle build configuration also?

Reply:
Indeed, adding the property works. Nice one!

Reply:
I'll revert back to assemble

Reply:
This means we also have to update the instructions for adapting an existing project to Quarkus. 
I changed the instruction too.

Reply:
I squashed the commits